src/test/rgw/sfs/test_rgw_sfs_wal_checkpoint.cc

```diff
@@ -78,7 +79,7 @@ class TestSFSWALCheckpoint : public ::testing::Test {
       size_t num_threads, size_t num_objects
   ) {
     std::atomic<std::uintmax_t> max_wal_size{0};
-    fs::path wal(test_dir / "s3gw.db-wal");
+    fs::path wal(test_dir / sqlite::DB_WAL_FILENAME);
```

Comment:
can we maybe define this as a function of DB_FILENAME, instead of having to keep a `#define` in the header solely for this test?

Reply:
That would leak an sqlite implementation detail that a reader does not need to know about. I'd rather have this